fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -98,31 +100,49 @@ public enum OlapTableState {
         WAITING_STABLE
     }
 
+    @SerializedName(value = "clusterId")
     protected int clusterId;
 
+    @SerializedName(value = "state")
     protected OlapTableState state;
 
     // index id -> index meta
+    @SerializedName(value = "indexIdToMeta")
     protected Map<Long, MaterializedIndexMeta> indexIdToMeta = Maps.newHashMap();
     // index name -> index id
+    @SerializedName(value = "indexNameToId")
     protected Map<String, Long> indexNameToId = Maps.newHashMap();
 
+    @SerializedName(value = "keysType")
     protected KeysType keysType;
+
+    @SerializedName(value = "partitionInfo")
     protected PartitionInfo partitionInfo;
+
+    @SerializedName(value = "idToPartition")
     protected Map<Long, Partition> idToPartition = new HashMap<>();
+
+    @SerializedName(value = "nameToPartition")
     protected Map<String, Partition> nameToPartition = Maps.newTreeMap(String.CASE_INSENSITIVE_ORDER);
 
+    @SerializedName(value = "defaultDistributionInfo")
     protected DistributionInfo defaultDistributionInfo;
 
     // all info about temporary partitions are save in "tempPartitions"
+    @SerializedName(value = "tempPartitions")
     protected TempPartitions tempPartitions = new TempPartitions();
 
     // bloom filter columns
+    @SerializedName(value = "bfColumns")
     protected Set<String> bfColumns;
+
+    @SerializedName(value = "bfFpp")
     protected double bfFpp;
 
+    @SerializedName(value = "colocateGroup")
```

Comment:
I recommend adding only the properties we need for now

Reply:
OK, I will remove the codes